st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -165,6 +166,9 @@ def _get_output_values(self, exit_code, stdout, stderr, timed_out):
         else:
             error = None
 
+        if "InvalidStatusException" in stderr:
```

Comment:
Anyway, let's get merged what we have right now, but keep an eye on this going forward (e.g. if a user reports an issue with it or similar).